### DISCUSSION TITLE
Rename dotnet to...

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/runtime#71459] [Ideas] [answered]
who'd like to rename Dotnet to other names?
DoyNet
JoyNet
DarkWeb
JobGet
SoyPat

**Comment by deeprobin** [comment]
1st april?

**Comment by teo-tsirpanis** [comment, marked as answer]
I find it quite unlikely that .NET will be renamed any time soon, and in the event it would happen, that new name would definitely not be one of these.
